tests/edge-cases/divmod-01.txt

```diff
@@ -0,0 +1,2 @@
+brun '(divmod)'
```

Comment:
@arvidn @richardkiss If we only return one value, we should consider changing the name `divmod` -> e.g. `div`

Reply:
This hasn't been changed yet, as we haven't come to a consensus on `div` vs `divmod`. Let me run a quick test.

Reply:
```
kiss@oasis ~ % python -m timeit 'A = 74738383914845; B = 83717484' "divmod(A,B)"
10000000 loops, best of 3: 0.164 usec per loop
kiss@oasis ~ % python -m timeit 'A = 74738383914845; B = 83717484' "A/B"        
10000000 loops, best of 3: 0.0503 usec per loop
kiss@oasis ~ % python -m timeit 'A = 74738383914845; B = 83717484' "A%B"
10000000 loops, best of 3: 0.0497 usec per loop
```

Okay, this is pretty wack. I don't know what to make of these results.

Reply:
It's almost as if it's spending most of its time making the tuple.

Reply:
Okay, let's use integer division.
```
kiss@oasis ~ % python -m timeit 'A = 74738383914845; B = 83717484' "A//B"       
10000000 loops, best of 3: 0.0522 usec per loop
kiss@oasis ~ % python -m timeit 'A = 74738383914845; B = 83717484' "A%B" 
10000000 loops, best of 3: 0.0512 usec per loop
kiss@oasis ~ % python -m timeit 'A = 74738383914845; B = 83717484' "(A//B, A%B)"
10000000 loops, best of 3: 0.117 usec per loop
kiss@oasis ~ % python -m timeit 'A = 74738383914845; B = 83717484' "divmod(A,B)"
10000000 loops, best of 3: 0.166 usec per loop
kiss@oasis ~ % python -m timeit 'A = 74738383914845; B = 83717484' "Q = A//B; (Q, A - B * Q)"
10000000 loops, best of 3: 0.104 usec per loop
```

Seems like calculating the remainder using arithmetic is the fastest. I recommend we add integer division and keep divmod. Your thoughts @aqk @arvidn @bramcohen ?

Reply:
I'm ok with keeping both, as long as div(A,B) always means exactly divmod(A,B)[0]